R/R/transformation.R

```diff
@@ -147,7 +148,8 @@ adstock_weibull <- function(x, shape, scale, windlen = length(x), type = "cdf")
     x_decayed <- x
     thetaVecCum <- 1
   }
-  return(list(x = x, x_decayed = x_decayed, thetaVecCum = thetaVecCum))
+  inflation <- sum(x_decayed) / sum(x)
```

Comment:
It's good that we have this inflation output here. but this is just the total inflation, while for each date the inflation will be different. Now in robyn_response, we already calculate the inflation for those metric_ds dates, which will be used in the allocator later to scale up the raw spend for those specific periods. therefore, maybe rename the inflation here as inflation_total?